docs/help/operator.md

```diff
@@ -735,3 +735,30 @@ spec:
       secrets:
       - "my-logging-config"
 ```
+
+## Configuring PodDisruptionBudget for broker deployment
+
+The ActiveMQArtemis custom resource offers a PodDisruptionBudget option
+for the broker pods deployed by the operator. When it is specified the operator
+will deploy a PodDisruptionBudget for the broker deployment.
+
+For example
+
+```yaml
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: broker
+  namespace: activemq-artemis-operator
+spec:
+spec:
+  deploymentPlan:
+    size: 2
+    image: placeholder
+    podDisruptionBudget:
+      minAvailable: 1
+```
+
+When deploying the above custom resource the operator will create a PodDisruptionBudget
+object with the **minAvailable** set to 1. The operator also sets the proper selector
```

Comment:
is the use case here that by default a deployment could be scaled to zero and we want to avoid that?

Reply:
The podDisruptionBudget is optional and it seems doesn't prevent the voluntary scale down (if user choose to do so by updating the CR).

Reply:
that makes sense, the user can still scale to 0 but kube won't do that via eviction if the budget disallows